Source/WebCore/dom/Element.cpp

```diff
@@ -3828,6 +3844,13 @@ AtomString Element::computeInheritedLanguage() const
     return document().contentLanguage();
 }
 
+AtomString Element::langForStyleResolution() const
+{
+    if (!hasRareData())
+        return nullAtom();
+    return elementRareData()->langForStyleResolution();
```

Comment:
shouldn't this return document element lang if not explicitly set?

Reply:
Fixed.